DevDashboard/cloud/landing/field-notes/index.html

```
@@ -0,0 +1,716 @@
+          <h2 class="font-display text-[clamp(2.1rem,5vw,3.4rem)] font-medium leading-[1.02] tracking-[-0.02em]">
+            Your machine. Your keys.<br/><span class="italic font-normal text-oxblood-soft">Your phone.</span>
+          </h2>
+          <p class="mt-5 max-w-md text-[15px] leading-relaxed text-cream/60">
+            Join the early access list. Self-host free today, or get an invite to the managed beta — both
+            ship with the open-source agent you can audit line by line.
+          </p>
+        </div>
+        <form class="flex flex-col gap-3" onsubmit="return false;">
+          <div class="rounded-full border border-cream/15 bg-cream/[0.06] p-1.5">
+            <div class="flex items-center gap-2">
+              <input type="email" required placeholder="you@yourmachine.dev" class="w-full bg-transparent px-4 py-2.5 text-[14px] text-cream placeholder:text-cream/35 focus:outline-none" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)